scoped_model_counter/pubspec.yaml

```diff
@@ -0,0 +1,24 @@
+name: scoped_model_counter
+description: >
+  The starter Flutter application, but using ScopedModel to manage state.
+
+version: 1.0.0+1
+
+environment:
+  sdk: ">=2.0.0-dev.68.0 <3.0.0"
```

Comment:
I guess that this comes from a template somewhere. Maybe it should be ">=2.0.0 <3.0.0"?

Reply:
This comes directly from the "new app" template. I'm gonna keep this for now.